cmd/main.go

```diff
@@ -281,6 +285,7 @@ func startServiceController(mgr manager.Manager, nsxClient *nsx.Client) {
 		networkpolicycontroller.StartNetworkPolicyController(mgr, commonService, vpcService)
 		service.StartServiceLbController(mgr, commonService)
 		subnetbindingcontroller.StartSubnetBindingController(mgr, subnetService, subnetBindingService)
+		inventory.StartupInventoryController(mgr, inventoryService, cf)
```

Comment:
Update the implementation of StartupInventoryController to the following to keep consistent with other controller? As you do not handle the error here, you may prefer to exit inside the controller upon error as other controllers do.

```
func StartInventoryController(mgr ctrl.Manager, service *inventory.InventoryService, cf *config.NSXOperatorConfig) {
    controller := NewInventoryController(mgr.GetClient(), service, cf)
    if err := controller.Start(mgr); err != nil {
        log.Error(err, "Failed to create Inventory controller", "controller", "Inventory")
	os.Exit(1)
    }
}

func (c * InventoryController) Start(mgr ctrl.Manager) error {
	return r.setupWithManager(mgr)
}
```

Reply:
OK.